src/load_Image/rotate.h

```diff
@@ -0,0 +1,32 @@
+#ifndef UNREAD__SRC__LOAD_IMAGE__ROTATE_H__
+#define UNREAD__SRC__LOAD_IMAGE__ROTATE_H__
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <gtk/gtk.h>
+#include <gdk/gdk.h>
+
+#include "tools.h"
+#include "../src/utils/bool.h"
```

Comment:
wong path, repace by 
```suggestion
#include "../utils/bool.h"
```